src/Pie.js

```diff
@@ -107,8 +106,8 @@ const Pie = ({ sections, radius, innerRadius, backgroundColor, strokeCap, divide
             radius={radius}
             width={width}
             color={color}
-            startAngle={startAngle + dividerSize}
-            arcAngle={arcAngle - dividerSize}
+            startAngle={startAngle + dividerSize / 2}
+            arcAngle={arcAngle - dividerSize / 2}
```

Comment:
the arcAngle should be `arcAngle={arcAngle - dividerSize}`, it's `endAngle - startAngle`

Reply:
Sorry for pushing back and forth, I think that's the last thing to launch this PR

Reply:
Doing `startAngle={startAngle + dividerSize / 2} arcAngle={arcAngle - dividerSize}` makes the sections appears slightly too small

Reply:
yes it will, the idea is that if all the section's total percentage if 100, for example [10, 20, 30, 40] which is [0-10, 10-30, 30-60, 60-100], then it will be rendered as [1-9, /, 11-29, /, 31-59, /, 61, 99, /]

Reply:
if we don't do that way, if will exceed 100 percent as total

Reply:
I put my rig away for the night, but to be clear you think it should be `startAngle={startAngle + dividerSize / 2} arcAngle={arcAngle - dividerSize}`?

I will try and check it tomorrow

Reply:
thank you so much @zgordon01 , I understand you idea about divider now, your divider simply add the total percentage of the sections, that would cause problem if the total percentage is 100, my idea is reduce the divider's size from each section, that would make the section a bit smaller than it should be, but I think we shouldn't change the position of each section, if we simply want to add the dividers between sections, we could add those dividers manually via sections, WDYT?

Reply:
Now I remember the code I removed in this commit https://github.com/nihgwu/react-native-pie/pull/32/commits/4dda11bb8b2250a6c3bfc5d66cb26e3505805d92 was to address showing a divider when the total percentage is `100` for a single slice. I can come up with something better for it :) (I can make any changes related to this just let me know)

The idea is to reduce the sections proportionally equal to the divider size. I think this solution is much better than adding dividers manually, as we also automatically handle round stroke cap which is very messy to do through the sections array.

Our use case in our app exactly matches the middle left image above which is impossible without a code change, so if you don't think this is a suitable addition we would have no choice but to fork this repo. Let me know your thoughts!

Reply:
> The idea is to reduce the sections proportionally equal to the divider size. I think this solution is much better than adding dividers manually, as we also automatically handle round stroke cap which is very messy to do through the sections array.

I think that's also my idea `my idea is reduce the divider's size from each section, that would make the section a bit smaller than it should be`

Reply:
seems one of my comments lost from the thread https://github.com/nihgwu/react-native-pie/pull/32#discussion_r364546607

Reply:
https://github.com/nihgwu/react-native-pie/pull/32#discussion_r364552878
We are reducing the sections proportionally, the problem is that the startAngle should be `startValue + dividerSize/2` and the arcAngle should be `arcAngle - dividerSize` as we reduce half of the divider's size from both start and end

Reply:
another way could be `startAngle = startValue` and `arcAngle = arcAngle - dividerSize`, we reduce the divider from the end only

Reply:
I'm  not saying we should render the dividers manually via sections, I just want to say that the divider's size should not be added to the total percentage, so the divider's size should be reduced from each section, or if we have [10, 20, 30, 40] and dividerSize = 2, the total percentage would be `10+2+20+2+30+2+40+2 = 108`, we should make it be `8+2+18+2+28+2+38+2 = 100`, that's all my arguments for this change

Reply:
@nihgwu please check again, I took your suggestion to use the below: 

> [#32 (comment)](https://github.com/nihgwu/react-native-pie/pull/32#discussion_r364552878)
> We are reducing the sections proportionally, the problem is that the startAngle should be `startValue + dividerSize/2` and the arcAngle should be `arcAngle - dividerSize` as we reduce half of the divider's size from both start and end

I also added back a check for use cases that have a 100% section

Reply:
If/when you merge, please do a squash merge because I have a lot of commits :)